keps/sig-apps/3939-allow-replacement-when-fully-terminated/README.md

```diff
@@ -700,6 +712,10 @@ This section must be completed when targeting alpha to a release.
     - kube-controller-manager (for main functionality)
     - kubelet (for supporting functionality: transition to phase=Failed)
 
+**After GA**
```

Comment:
nit: feel free to remove all those additions.

locking FG at GA is a standard thing and there is no need to update the KEP to reflect that

[and it's actually easier to follow the doc without those]

Reply:
removed